docs/capabilities.md

```diff
@@ -0,0 +1,40 @@
+
+The software identity of a receiver can be determined by its USB id (reported by Solaar and also viewable in Linux using `lsusb`).  The software identity of a device can be determined by its Wireless PID as reported by Solaar.  
+
+Even something as fundamental as pairing works differently for different receivers.  For Unifying receivers, pairing adds a new paired device, but only if there is an open slot on the receiver.  So these receivers need to be able to unpair devices that they have been paired with or else they will not have any open slots for pairing.  Some other receivers, like the receiver with USB ID `046d:c532`, can only pair with particular kinds of devices and pairing a new device replaces whatever device of that kind was previously paired to the receiver.  These receivers cannot unpair.  Further, some receivers can pair an unlimited number of times but others can only pair a limited number of times.
+
+Only some connections between receivers and devices are possible.  In should be possible to connect any device with a Unifying logo on it to any receiver with a Unifying logo on it.  Receivers without the Unifying logo probably can connect only to the kind of devices they were bought with and devices without the Unifying logo can probably only connect to the kind of receiver that they were bought with.
+
+
+## Supported features
+
+Solaar uses the HID++ protocol to pair devices to receivers and unpair devices from receivers.  Solaar also uses the HID++ protocol to display features of receivers and devices.  Solaar can modify some of the features of devices.  Solaar currently only displays some features and can modify even fewer.
+
+Solaar does not do anything beyond using the HID++ protocol to change the behavior of receivers and devices.  In particular, Solaar cannot change how the operating system turns the keycodes that a keyboard produces into characters that are sent to programs.  That is the province of HID device drivers and other software (such as X11).
+
+Logitech receivers and devices have firmware in them.  Some of the firmware can be updated using Logitch software in Windows.  For example, there are security issues with some Logitech receivers and devices and Logitech has firmware updates to alleviate some of these problems.  Some Logitech firmware can also be updated in Linux using `fwupdmgr`.  WARNING: Updating firmware can cause a piece of hardware to to become permanently non-functional if something goes wrong with the update or the update installs the wrong firmware.
```

Comment:
Typo: `Logitch` -> `Logitech.

"some of these problems" -> "these problems"?

Reply:
OK